bldsys/cmake/template/sample_api/CMakeLists.txt.in

```diff
@@ -0,0 +1,27 @@
+# Copyright (c) 2019-2023, Arm Limited and Contributors
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 the "License";
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+get_filename_component(FOLDER_NAME ${CMAKE_CURRENT_LIST_DIR} NAME)
+get_filename_component(PARENT_DIR ${CMAKE_CURRENT_LIST_DIR} PATH)
+get_filename_component(CATEGORY_NAME ${PARENT_DIR} NAME)
+
+add_sample(
+    ID ${FOLDER_NAME}
+    CATEGORY ${CATEGORY_NAME}
+    AUTHOR "Khronos"
+    NAME "@SAMPLE_NAME@"
+    DESCRIPTION "Sample description")
```

Comment:
It is not present in current template but in CMAKEList the shader files should be listed.
Since loading them is present in sample.cpp.in I would suggest and add creation of both files in path:
shader/@SAMPLE_NAME@/triangle.vert
shader/@SAMPLE_NAME@/triangle.frag
would it be a big effort to add it in the script?

```suggestion
    DESCRIPTION "Sample description"
    "@SAMPLE_NAME@/triangle.vert"
    "@SAMPLE_NAME@/triangle.frag")
```
alternatively it could 
```
    DESCRIPTION "Sample description"
    "triangle.vert"
    "triangle.frag")
```

Reply:
the second option seems more reasonable since there are samples without shader files

Reply:
In other samples I saw that it was done like this:
    DESCRIPTION "Sample description"
    SHADER_FILES_GLSL
        "triangle.vert"
        "triangle.frag")
        
So I did the same.